### PR TITLE
TRIVIAL: adding version 1.4 to config and redirects

### PR DESCRIPTION
**Pull request by Mara3l**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: docs/layouts/index.redir [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-/ {{ .Site.BaseURL }}/1.3/ 301!
-/latest/ {{ .Site.BaseURL }}/1.3/ 301
-/docs/ {{ .Site.BaseURL }}/1.3/ 301!
+/ {{ .Site.BaseURL }}/1.4/ 301!
+/latest/ {{ .Site.BaseURL }}/1.4/ 301
+/docs/ {{ .Site.BaseURL }}/1.4/ 301!
+/1.4.0/ {{ .Site.BaseURL }}/1.4 301!
 /1.3.0/ {{ .Site.BaseURL }}/1.3 301!
 /1.2.0/ {{ .Site.BaseURL }}/1.2 301!
-/1.3.1.dev1 {{ .Site.BaseURL }}/dev 301!
```
Comment (Mara3l) [inline docs/layouts/index.redir:7]: No longer on dev

**Review by hkad98 — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR301

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul/115/ : FAILURE in 1m 47s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/265/ : SUCCESS in 50s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/268/ : SUCCESS in 10s

**Pull request merged**